package.json

```diff
@@ -184,6 +184,7 @@
         "babel-plugin-react-remove-properties": "^0.3.0",
         "babel-plugin-rewire": "^1.0.0",
         "babel-preset-react-app": "^9.1.0",
+        "box-annotations": "^3.13.0",
```

Comment:
Do we want to add this right now?

Reply:
I figured we could add now in the story, or provide instructions to develop on Annotations.

Reply:
I can remove it if you think it isn't a good idea to include yet.

Reply:
I don't think we should include it right now. We can each just add it locally, I think.